circle.yml

```diff
@@ -92,6 +92,53 @@ jobs:
       - build
       - test
 
+  clang-latest-coverage:
+    executor: linux-clang-9
+    environment:
+      BUILD_TYPE: Coverage
+      TESTS_FILTER: unittests
+    steps:
+      - run:
+          name: "Install lcov"
+          working_directory: /tmp
+          command: |
+            sudo apt-get -q update
+            # sudo apt-get install -qy lcov --no-install-recommends
+            apt-get download lcov
+            sudo dpkg -i --force-depends lcov*.deb
```

Comment:
why this unusual way to install?

Reply:
Although being just a perl script, the lcov debian package depends on gcc package. Installing in a normal way will pull additional 300 MB. I plan to add lcov to the cpp-build-env image, probably downloading official release from http://ltp.sourceforge.net/coverage/lcov.php.

The trick used here is probably not good for docker images as apt-get will start reporting broken packages installed.